src/sentry/search/events/fields.py

```diff
@@ -2408,10 +2475,16 @@ def resolve_function(self, function: str, match: Optional[Match[str]] = None) ->
         snql_function = self.function_converter.get(name)
 
         arguments = snql_function.format_as_arguments(name, arguments, self.params)
+        for arg in snql_function.args:
+            if type(arg) == SnQLColumn:
+                arguments[arg.name] = self.column(arguments[arg.name])
 
         if snql_function.snql_aggregate is not None:
             self.aggregates.append(snql_function.snql_aggregate(arguments, alias))
-        return snql_function.snql_aggregate(arguments, alias)
+            return snql_function.snql_aggregate(arguments, alias)
+
+        if snql_function.snql_column is not None:
+            return snql_function.snql_column(arguments, alias)
```

Comment:
Can we assert that this function always returns in one of these conditions and that it should never fall off and return `None`?

Reply:
We validate this [here](https://github.com/getsentry/sentry/pull/27782/files#diff-e7d5c143e6b668a98f4dc22ec4fdc3796c413d23cfbd4d3d43c1cd001465cddeR2170) already, so I just removed the if statement and just return `snql_function.snql_column(...)`